ray/adaptdl_ray/adaptdl/utils.py

```diff
@@ -34,15 +35,19 @@ def pgf_to_allocation(pgf) -> List[str]:
     return allocs
 
 
-def allocation_to_pgf(alloc: List[str]):
+def allocation_to_pgf(alloc: List[str], resources_per_node=None):
```

Comment:
Instead of `None`, maybe initialize with `{"CPU" = 1.0}`? so the `if not` block can be removed.

Reply:
Python doesn't do object as defaults well (there is only one object per that function globally)